data/docs/instrumentation/opentelemetry-quarkus.mdx

```diff
@@ -0,0 +1,419 @@
+    <figcaption><i>OpenTelemetry helps generate and collect telemetry data from Quarkus applications which can then be sent to SigNoz for storage, visualization, and analysis.</i></figcaption>
+</figure>
+
+After completing this guide, you will have:
+
+- A Quarkus application sending traces, metrics, and logs to SigNoz
+- Real-time visibility into your application's performance
+- Ability to troubleshoot issues using distributed tracing
+- Monitoring dashboards for your application metrics
+
+Let's understand how to install and run OpenTelemetry in your Quarkus application.
+
+## Requirements
+
+- A Quarkus REST application with basic endpoints.
+- Java 17 or higher environment
+- Docker (if using Docker and Kubernetes)
+- OpenTelemetry extension added to your project:
+
+```bash
+# Using Quarkus CLI
+quarkus extension add opentelemetry
+```
+
+```bash
+# Using Maven command
+./mvnw quarkus:add-extension -Dextensions="opentelemetry"
+```
+
+```bash
+# Using gradle, add the following to your build.gradle file
+implementation 'io.quarkus:quarkus-opentelemetry:3.17.2'
+```
+
+## Configuration Steps
+
+ - Update `application.properties`
+
+```properties:src/main/resources/application.properties
+# Application name and version
+quarkus.application.name=quarkus-app
+quarkus.application.version=1.0.0
+
+# Ingestion key for authentication(Only if using Signoz Cloud)
+quarkus.otel.exporter.otlp.headers=${OTEL_EXPORTER_OTLP_HEADERS:}
+
+# Enable OpenTelemetry
+quarkus.otel.sdk.disabled=false
+
+# Traces Configuration
+quarkus.otel.traces.enabled=true
+quarkus.otel.traces.sampler=parentbased_always_on
+quarkus.otel.traces.sampler.arg=1.0
+quarkus.otel.traces.exporter=otlp
+
+# OTLP Exporter Configuration
+quarkus.otel.exporter.otlp.endpoint=${OTEL_EXPORTER_OTLP_ENDPOINT}
+quarkus.otel.exporter.otlp.traces.endpoint=${OTEL_EXPORTER_OTLP_ENDPOINT}/v1/traces
+quarkus.otel.exporter.otlp.protocol=http/protobuf
+
+# Metrics Configuration
+quarkus.otel.metrics.enabled=true
+quarkus.otel.exporter.otlp.metrics.endpoint=${OTEL_EXPORTER_OTLP_ENDPOINT}/v1/metrics
+quarkus.otel.metrics.exporter=otlp
+
+# Logging Configuration
+quarkus.otel.logs.enabled=true
+quarkus.otel.exporter.otlp.logs.endpoint=${OTEL_EXPORTER_OTLP_ENDPOINT}/v1/logs
+quarkus.otel.logs.exporter=otlp
+
+quarkus.log.level=INFO
+quarkus.log.category."io.opentelemetry".level=INFO
+
+# Console logging with trace context
+quarkus.log.console.format=%d{HH:mm:ss} %-5p traceId=%X{traceId}, parentId=%X{parentId}, spanId=%X{spanId} [%c{2.}] (%t) %s%e%n
+
+# File logging with trace context (optional)
+quarkus.log.file.enable=true
+quarkus.log.file.path=logs/application.log
+quarkus.log.file.format=%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p traceId=%X{traceId}, parentId=%X{parentId}, spanId=%X{spanId} [%c{3.}] (%t) %s%e%n
+
+# Resource Attributes
+quarkus.otel.resource.attributes=${OTEL_RESOURCE_ATTRIBUTES}
+```
+
+<Tabs>
+<TabItem value="cloud" label="SigNoz Cloud" default>
```

Comment:
Just below this tab, let's add an Info admonition saying:

<Admonition type="info">
If you haven't already, signup for [SigNoz Cloud](https://signoz.io/teams/).
</Admonition>